sdk/formrecognizer/ai-form-recognizer/README.md

```diff
@@ -1,13 +1,12 @@
 # Azure Form Recognizer client library for JavaScript
 
-Azure Cognitive Services [Form Recognizer](https://azure.microsoft.com/services/cognitive-services/form-recognizer/) is a cloud service that uses machine learning to recognize text and table data
-from form documents. It includes the following main functionalities:
+Azure Cognitive Services [Form Recognizer](https://azure.microsoft.com/services/cognitive-services/form-recognizer/) uses cloud-based machine learning to extract structured data from form documents. Its features include:
 
-* Custom models - Recognize field values and table data from forms. These models are trained with your own data, so they're tailored to your forms. You can then take these custom models and recognize forms. You can also manage the custom models you've created and see how close you are to the limit of custom models your account can hold.
-* Content API - Recognize text and table structures, along with their bounding box coordinates, from documents. Corresponds to the REST service's Layout API.
-* Prebuilt receipt model - Recognize data from sales receipts using a prebuilt model.
+- Custom Form Models: Create machine learning models to recognize structured data such as field values and tables from forms. The models are trained on your own data, so they're tailored to your forms' structure. Then, use the custom models to extract recognized fields and content from forms. Use the API to manage your models by creating, listing, deleting, and copying models;
+- Content API: Extract raw page elements&mdash;such as text words/lines, tables, and selection marks&mdash;and their bounding box coordinates from documents (this corresponds to the REST service's "layout" API).
+- Prebuilt Models: Extract data from certain types of common documents (such as receipts, invoices, and business cards) using models developed by the Azure Form Recognizer team.
 
-**Note:** This package targets Azure Form Recognizer service API version 2.0.
+**Note:** This package targets Azure Form Recognizer service API version 2.x.
```

Comment:
do we not want to say 2.1-preview?

Reply:
I figure it's just another thing to change every release. I guess the way I think about it is that we are currently targeting this API series, to differentiate from older SDKs that targeted 1.x. Especially since the only GA package is 2.0 for now. For us in JS we really only target one per package, but doesn't Python support multi-API?